v2/pkg/engine/resolve/response.go

```diff
@@ -23,13 +23,20 @@ type GraphQLSubscriptionTrigger struct {
 	QueryPlan      *QueryPlan
 }
 
+// GraphQLResponse contains intermediate and final data of the response.
+// Fields are filled in this order:
+//
+//  1. Planner fills RawFetches and Info fields.
```

Comment:
Planner creates RawFetches, Info, and Data

Reply:
done